Gems/Atom/Tools/AtomToolsFramework/Code/Source/Util/Util.cpp

```diff
@@ -82,7 +84,9 @@ namespace AtomToolsFramework
             return QFileInfo();
         }
 
-        return selectedFileInfo;
+        // Reconstructing the file info from the absolute path and expected extension to compensate for an issue with the save file
+        // dialog adding the extension multiple times if it contains "." like *.lightingpreset.azasset
+        return QFileInfo(selectedFileInfo.absolutePath() + AZ_CORRECT_FILESYSTEM_SEPARATOR_STRING + selectedFileInfo.baseName() + "." + initialExt);
```

Comment:
suggestion: This is just the sort of thing that would be fantastic to test 😄 Have a number of `initialPath` versions and verify we get the expected outcome. It might be necessary to refactor line 67 to remove the Qt dependency but this could be very helpful to validate future changes.